submissions/html and css tasks

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```

Comment:
Please split this  file into HTML and CSS files with the correct file extensions.
It is not clear what this file does.

Reply:
Please how do i do that, i get 404 error whenever i try

Reply:
The name of the only file in this PR is  `html tasks`. It has no extensions, so it will not be interpreted correctly.

First create a folder with your name in the `submissions` folder
Your submission should look like this
submissions/
       YourNameDominic/
             html/
                   filename.html
                   filename.css

PS: How did you view this file on your computer?